lib/Runtime/Base/CallInfo.h

```diff
@@ -91,6 +91,11 @@ namespace Js
 
         static ArgSlot GetArgCountWithoutExtraArgs(CallFlags flags, ArgSlot count);
 
+        static bool HasExtraArg(CallFlags flags)
+        {
+            return (flags & CallFlags_ExtraArg) || CallInfo::HasNewTarget(flags);
```

Comment:
I would recommend changing this slightly to something like

```cpp
Assert(!CallInfo::HasNewTarget(flags) || flags & CallFlags_ExtraArg);
return (flags & CallFlags_ExtraArg);
```

Reply:
I will add the assert, but keep the implementation. - Just to be sure we do not end up with a buffer overrun if condition is somehow violated in Release.